truss/remote/baseten/remote.py

```diff
@@ -213,7 +214,7 @@ def sync_truss_to_dev_version_by_name(
         # thread to keep it alive. When this loop is interrupted by the user, then the whole process
         # can shutdown gracefully.
         while True:
-            pass
+            time.sleep(100)
```

Comment:
could you add a comment w/ TODO to refactor this path to run on the main thread?